pkg/vault/contracts/Vault.sol

```diff
@@ -1567,6 +1574,16 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
         return _implementation();
     }
 
+    /// @inheritdoc IVaultMain
+    function getMinimumTradeAmount() external view returns (uint256) {
+        return _MINIMUM_TRADE_AMOUNT;
+    }
+
+    /// @inheritdoc IVaultMain
+    function getMinimumWrapAmount() external view returns (uint256) {
+        return _MINIMUM_WRAP_AMOUNT;
+    }
```

Comment:
Hm; I've thought about this.

How much of the bytecode size change is because of these getters? I think we can't afford them unfortunately.
Maybe I'd move the immutables to the base storage layer and the getters to VaultAdmin, and then set them in the constructor of the vault extension and the main vault. It's a bit more complex (at the constructor levels at least), but it will save some bytes that we need here.

Reply:
I tried that though - there seems to be an issue doing this with immutables (vs. regular storage, like all the other times we do this). I will try again; maybe I was missing something.